s3/src/test/scala/akka/stream/alpakka/s3/impl/S3NoMock.scala

```diff
@@ -16,26 +14,25 @@ import org.scalatest.{BeforeAndAfterAll, FlatSpecLike, Matchers}
 import scala.concurrent.Await
 import scala.concurrent.duration._
 
-/**
- *
- * @author siva
- */
 class S3NoMock extends FlatSpecLike with BeforeAndAfterAll with Matchers with ScalaFutures {
 
   implicit val actorSystem = ActorSystem()
   implicit val materializer = ActorMaterializer()
 
   val bucket = "test-bucket"
   val objectKey = "test"
-  val objectValue = "Some content."
+
+  val objectValue = "Some String"
+  val metaHeaders: Map[String, String] = Map("location" -> "Africa", "datatype" -> "image")
 
   it should "upload with real credentials" ignore {
 
     val source: Source[ByteString, Any] = Source(ByteString(objectValue) :: Nil)
+    //val source: Source[ByteString, Any] = FileIO.fromPath(Paths.get("/tmp/IMG_0470.JPG"))
 
-    val result = source.runWith(S3Client().multipartUpload(bucket, objectKey, contentType = ContentTypes.`text/plain(UTF-8)`))
+    val result = source.runWith(S3Client().multipartUpload(bucket, objectKey, metaHeaders = MetaHeaders(metaHeaders)))
 
-    val multipartUploadResult = Await.ready(result, 5.seconds).futureValue
+    val multipartUploadResult = Await.ready(result, 90.seconds).futureValue
```

Comment:
just curious, what is the materialized value (result)? A Future of CompleteMultipartUploadResult? Looks like a Future[Future] here?

Is 90 seconds realistic (needed)?